### DISCUSSION TITLE
小白提问如何在volantis下载aplayer并应用到hexo博客上

### DISCUSSION THREAD

**Discussion by HanDaWei236** [volantis-x/hexo-theme-volantis#820] [Q&A] [answered]
![](https://i0.hdslb.com/bfs/album/4ee9fa44ffbf350d13d77b5e51ea1c178a116bfa.png)
![image](https://user-images.githubusercontent.com/115543788/197000438-f3e4fe4e-b519-4e21-ae4b-783fcf1588bb.png)
下载aplayer并插入代码后我的博客依旧没有显示音乐播放器，请问这是怎么回事

**Comment by BeaCox** [comment, marked as answer]
如果你想要音乐播放器吸附在页脚左侧，如图
![](https://kjimg10.360buyimg.com/ott/jfs/t1/219771/15/22444/89971/63520e11E4069ad05/1baaf27d9d1790a5.png)
请在`_config.volantis.yml`设置
```yml
  aplayer:
    ...
    fixed:true
```
并设置
```yml
site_footer:
  layout: [......, aplayer]
```

**Reply by BeaCox** [reply]
如果要出现在页脚，请设置fixed为false，并如上配置site_footer
如果要出现在侧边栏，参考[音乐部件配置](https://volantis.js.org/v6/theme-settings/#%E9%9F%B3%E4%B9%90%E9%83%A8%E4%BB%B6)
如果要出现在导航栏，参考[导航栏配置](https://volantis.js.org/v6/theme-settings/#%E6%92%AD%E6%94%BE%E5%99%A8)

更多问题，文档以及主题仓库配置文件搜索关键词：播放器/音乐/aplayer
